tests/decodecorpus.c

```diff
@@ -1339,6 +1601,7 @@ static void advancedUsage(const char* programName)
     DISPLAY( "\n");
     DISPLAY( "Advanced arguments :\n");
     DISPLAY( " --content-size    : always include the content size in the frame header\n");
+    DISPLAY( " --use-dict #      : include a dictionary used to decompress the corpus\n");
```

Comment:
Prefer `--use-dict=#`, for consistency with other tools within `zstd` repository.
Look into `zstdcli.c` for guidance if need be.